client/app/dashboard/model/list/ModelList.js

```diff
@@ -830,8 +842,19 @@ angular.module('dashboard.Dashboard.Model.List', [
     }
     
   }
+
+  // reset grid when clicking all users
+  $scope.$on('$locationChangeSuccess', function() {
```

Comment:
This is causing the controller to load multiple times and causing the sortInfo to load the previous values on the query string